tests/coalaTest.py

```diff
@@ -179,3 +179,19 @@ def test_logged_error_causes_non_zero_exitcode(self):
             )
 
             assert exitcode == 1
+
+    def test_coala_no_color(self):
+        configure_logging(color=False)
+        with bear_test_module(), \
+                prepare_file(['#fixme'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                             coala.main,
+                             'coala', '-c', os.devnull,
+                             '-f', re.escape(filename),
+                             '-b', 'LineCountTestBear')
+            self.assertIn('This file has 1 lines.',
+                          stdout,
+                          'The output should report count as 1 lines')
```

Comment:
Up to you:
Maybe we want to be a bit more restrictive here? We know that the non-colored output has no special characters so we may assert for the complete message. But I would allow this too^^